dev-util/cmocka/files/cmocka-1.0.1_fix_32bit_librt.so.patch

```diff
@@ -0,0 +1,41 @@
+--- a/ConfigureChecks.cmake
++++ b/ConfigureChecks.cmake
+@@ -100,11 +100,9 @@ else (WIN32)
+ endif (WIN32)
+ 
+ find_library(RT_LIBRARY rt)
+-if (RT_LIBRARY)
+-    set(CMAKE_REQUIRED_LIBRARIES ${RT_LIBRARY})
+-endif (RT_LIBRARY)
+-
+-set(CMOCKA_REQUIRED_LIBRARIES ${CMAKE_REQUIRED_LIBRARIES} CACHE INTERNAL "cmocka required system libraries")
++if (RT_LIBRARY AND NOT LINUX)
++    set(CMOCKA_REQUIRED_LIBRARIES ${RT_LIBRARY} CACHE INTERNAL "cmocka required system libraries")
++endif ()
+ 
+ # OPTIONS
+ check_c_source_compiles("
+@@ -124,9 +122,10 @@ int main(void) {
+ endif(WIN32)
+ 
+ if (HAVE_TIME_H AND HAVE_STRUCT_TIMESPEC AND HAVE_CLOCK_GETTIME)
+-    set(CMAKE_REQUIRED_LIBRARIES ${RT_LIBRARY})
++    if (RT_LIBRARY)
++        set(CMAKE_REQUIRED_LIBRARIES ${RT_LIBRARY})
++    endif()
+ 
+-    message(STATUS "CMAKE_REQUIRED_INCLUDES=${CMAKE_REQUIRED_INCLUDES} CMAKE_REQUIRED_LIBRARIES=${CMAKE_REQUIRED_LIBRARIES}")
+     check_c_source_compiles("
+ #include <time.h>
+ 
+@@ -137,7 +136,10 @@ int main(void) {
+ 
+     return 0;
+ }" HAVE_CLOCK_GETTIME_REALTIME)
++
++    # reset cmake requirements
+     set(CMAKE_REQUIRED_INCLUDES)
++    set(CMAKE_REQUIRED_LIBRARIES)
+ endif ()
+ 
+ # ENDIAN
```

Comment:
Please use original upstream git patch with meta information. In general we prefer '-' instread of  '_'.